doc/source/configuration/wazuh.rst

```diff
@@ -2,6 +2,21 @@
 Wazuh
 =====
 
+The short version
```

Comment:
Is this intended as an introduction, or to be complete, for people familiar with the procedure?

Reply:
Intended as a short version for people who are familar with it but perhaps need a reminder.